app/serializers/skill.js

```diff
@@ -0,0 +1,4 @@
+import DS from 'ember-data';
+
+export default DS.RESTSerializer.extend({
```

Comment:
we can REAAAALLY clean this up by making it a single line:

export default DS.RESTSerializer.extend({ });